### ISSUE TITLE
Question regarding circBase search by COMPSRA

### ISSUE THREAD

**Issue by zntaylgit** [cougarlj/COMPSRA#58]
How exactly does COMPSRA determine if a read maps to a circRNA? Does it specifically consider only reads that map to the splice junction? Otherwise, reads from linear mRNAs that map to an exonic sequence in the circRNA may erroneously be attributed to circular molecules, correct? Basically, I'm asking how the circBase database is employed in COMPSRA.

**Comment by hadasvolk** [comment]
@zntaylgit have you ever figured this out? I'm asking myself the same question

**Comment by zntaylgit** [comment]
@hadasvolk No, not yet; I've been working on other things. I haven't gotten a reply yet. If there is a way to run Compsra while saving each identified feature's reads, I was going use that and then blast all reads to a complete circRNA sequence to see if the splice junction was the only portion covered to any read depth. But its on the back burner, and I was hoping the author would save me the trouble by replying here. They are likely busy too though so I'm just sitting tight for the moment.

**Comment by hadasvolk** [comment]
@zntaylgit I think the code does not attempt to diffrentiate the mRNA and circRNA. I wonder why or where this claim that all those reads are circRNA originated from?!

**Comment by zntaylgit** [comment]
@hadasvolk I'm not sure. Do they have code to create a replacement pre-built circRNA database I wonder?

**Comment by hadasvolk** [comment]
@zntaylgit haven't searched as deep, but all competing tools do not assign as much reads to circs. As far as I see it, read assignment to circ should be BSJ backed, otherwise it is only a cDNA match

**Comment by zntaylgit** [comment]
@hadasvolk well thats dissapointing. Have you also looked at the output of any of the other small RNAs counts from compsra relative to other tools? Looking at piRBank it seems that the the database lists boundaries of individual piRNAs, and the same is true for piRBase so I dont think there's a problem with those counts. But I agree, they must have  overlooked the fact its the splice junction alone that matters when counting circRNAs.

**Comment by zntaylgit** [comment]
@hadasvolk I should also ask, while I'm thinking about it, if you have looked at other circRNA prediction tools, would you recommend a particular circRNA prediction tool for its ability to take similar input as Compsra? In the near future I do want to investigate circRNA in my data too.

**Comment by hadasvolk** [comment]
@zntaylgit I did try the other relatives, but I think they suffer similar issues. 
If you refer to smRNA Seq data, I ended up doing all manually. Cleared t, r, pi, nc RNA and cDNA with bowtie2, remaining reads were aligned to miRBase with bowtie1. Whatever left including cDNA matches I fed to circexplorer2.
Maybe I should publish the pipeline ;)
